cogs/help.py

```diff
@@ -46,6 +46,7 @@
     "🤚": "Help Needed",
     "🔥": "Active",
     "⏳": "Stalled",
+    "🛠️": "Development",
```

Comment:
I feel this should be `Development Version` as just `Development` could be misleading